diskcache/core.py

```diff
@@ -2450,3 +2454,34 @@ def reset(self, key, value=ENOVAL, update=True):
 
         setattr(self, key, value)
         return value
+
+if hasattr(os, 'register_at_fork'):
+    _lock = threading.RLock()
+
+    def _acquireLock():
+        global _lock
+        if _lock:
```

Comment:
Why the check for `if _lock:`? Seems like it must exist by the time the function is defined.

Reply:
Sorry for the delay, was on an extended new-year's vacations.
Fixed.